www/topical-software.rst

```diff
@@ -90,6 +90,7 @@ Working environments
 - `IPython <http://ipython.org>`__: an interactive environment with many features geared towards efficient work in typical scientific usage. It borrows many ideas from the interactive shells of Mathematica, IDL, Matlab and similar packages. It includes special support for the matplotlib and gnuplot plotting packages. IPython also has support for (X)Emacs, to be used as a full IDE with IPython as the interactive Python shell.
 - `Jupyter Notebook <http://jupyter.org/>`__: a web application that works off of the IPython and allows you to create and share documents containing live code and can be used in all sorts of contexts from statistical modeling to data cleaning.
 - `Spyder <https://pythonhosted.org/spyder/>`__: A Qt based IDE suited to developing scientific applications.  Includes integrated and external Python consoles, code checking built into the editor, a graphical class browser and full support for matplotlib graphs.
+- `Microsoft Visual Studio <https://www.visualstudio.com/>`__: A free, rich, IDE that now natively supports Python, and Anaconda distribution. CPython, IronPython, the IPython REPL, Debugging, Profiling, Git, GitHub, are also supported. Also has a lighter version, named Code, which is a code editor with debugger. Both feature powerful IntelliSense, and support Windows and macOS, plus Linux in case of Code.
```

Comment:
Nice, licensing terms seem to have been improved recently!

A few nitpicks:
- ``rich, IDE `` --> remove the comma there
- ``Python, and Anaconda distribution`` --> remove comma and "distribution"
- ``Debugging, Profiling,`` --> not names so lowercase
- ``Git, GitHub, are`` --> `` Git and GitHub are``
- ``named Code `` --> ``name Visual Studio Code``
- ``macOS`` --> ``OS X``

Reply:
Okay, done. Thank you for the suggestions. Feel free to let me know if there's something else.